coffeenet-autoconfigure/src/main/java/rocks/coffeenet/autoconfigure/security/config/DevelopmentCoffeeNetWebSecurityConfigurerAdapter.java

```diff
@@ -52,7 +52,7 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth.inMemoryAuthentication()
             .withUser("admin")
             .password("admin")
-            .roles("COFFEENET-ADMIN", "USER")
+            .roles("COFFEENET-ADMIN", "USER", "SYSADMIN")
```

Comment:
I do not think that is a good idea :) That is very synyx specific.

Reply:
I'm not sure if COFFEENET-ADMIN is equal to SYSADMIN, but you are right it is only for testing purpose.

Reply:
Why should the coffeenet know any company specific roles?